package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Code for www.musicparsed.com",
   "scripts": {
     "postinstall": "webpack -r babel-register-ts -p",
-    "watch": "webpack -r babel-register-ts --watch --progress -d"
+    "watch": "webpack -r babel-register-ts --watch --progress -d",
+    "test": "mocha -r ts-node/register ts/tests/*.ts"
```

Comment:
No need for `ts-node`, we can just use `babel-register-ts` again. `babel-register-ts` is faster because it doesn’t check types (we’re already doing that with `tsc`).